src/Player/PlayerProxy.h

```diff
@@ -59,6 +110,7 @@ class PlayerProxy
     int getStatus();
     uint64_t getLiveSecs();
     uint64_t getRePullCount();
+    TranslationInfo getTranslationInfo();
```

Comment:
`TranslationInfo` updated just once when play successed, and some information(such as `BytesSpeed`) had not yet been generated at that time.

Reply:
I can make a flag that bytespeed has been received
But if it was not received, then now its value will be == -1

Reply:
Ok, I'm just want tell you that some real-time statistical information cannot be obtained by this function